### PR TITLE
Add HWiNFO 6.26

### PR DESCRIPTION
**Pull request by xiaoyang-sde**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/719)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
Duplicate: #985 and #952

File: manifests/REALiX/HWiNFO/6.26.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Id: REALiX.HWiNFO
+Publisher: HWiNFO
+Name: REALiX
```
Comment (OfficialEsco) [inline manifests/REALiX/HWiNFO/6.26.yaml:3]:
```suggestion
Publisher: REALiX
Name: HWiNFO
```

File: manifests/REALiX/HWiNFO/6.26.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Tags: CPU, GPU, HWiNFO, Monitor, Hardware
+Description: Comprehensive Hardware Analysis, Monitoring and Reporting for Windows and DOS.
+Homepage: https://www.hwinfo.com/
+Installers:
+  - Arch: x64
+    Url: https://www.sac.sk/download/utildiag/hwi_626.exe
+    Sha256: B76AAAEDB2A6DE4968AE8DA58B3EE3230DA2A6E525544D4168657BBEF07CBC9E
+    Switches:
+      Silent: "/quiet"
+      SilentWithProgress: "/passive"
```
Comment (OfficialEsco) [inline manifests/REALiX/HWiNFO/6.26.yaml:17]:
```suggestion
      Silent: /VERYSILENT
      SilentWithProgress: /VERYSILENT
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Publisher and Name was the wrong way around, alternatively publisher should be Martin Malik - REALiX.

/VERYSILENT is the correct silent install command.

File: manifests/REALiX/HWiNFO/6.26.yaml (hunk 1 above)
Comment (Berkeley-Reject) [reply manifests/REALiX/HWiNFO/6.26.yaml:3]: Thank you! I will correct it as soon as possible.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**